docs/specifications/tests/Methods.md

```diff
@@ -15,46 +15,90 @@ Obtain the parent zone of the given domain (zone).
 
 1. The given domain is assumed to be apex of a zone with the same
    name, and is here refered to as the _child zone_.
-2. A recursive lookup for the SOA record of the child domain name starting from the
+2. A recursive lookup for the SOA record of the child zone starting from the
    root zone is done, and the steps of the process are recorded.
 3. If the lookup reaches a name server that responds with a redirect (delegation)
-   directly to the requested child zone the method
-   succeeds. The zone in which the delegation was found is
-   considered to be the parent zone.  
+   directly to the requested child zone:
+
+   3.1 The method succeeds.
```

Comment:
Have you considered using proper lists? That would increase the spacing between the numbering and the content, making it easier on the eye.